port/stmicro/stm32/src/hals/STM32F103/backup.zig

```diff
@@ -0,0 +1,90 @@
+const microzig = @import("microzig");
+const backup_domain_protection = microzig.hal.power.backup_domain_protection;
+
+const bkp = microzig.chip.peripherals.BKP;
+pub const ASOS = microzig.chip.types.peripherals.bkp_v1.ASOS;
+pub const TPAL = microzig.chip.types.peripherals.bkp_v1.TPAL;
+
+const bkp_addr = @intFromPtr(bkp);
+const BKPData = packed struct(u32) {
+    data: u16,
+    __pad: u16 = 0,
```

Comment:
we just call these `paddingN` where N is the lowest bit so this would be `padding16`